src/mix/api/utils/entities-helpers.ts

```diff
@@ -10,8 +10,29 @@ import makeDebug from 'debug'
 
 const debug = makeDebug.debug('mix:utils:entities-helpers')
 
+export const buildConvertEntityBody = (params: any) => {
+  debug('buildConvertEntityBody()')
+
+  const {
+    hasA,
+    isA,
+    newType,
+    pattern,
+  } = params
+
+  const adjustedNewtype = newType.toUpperCase().replace('-', '_')
+
+  return {
+    ...(hasA === undefined ? {} : {entities: {hasA}}),
+    ...(isA === undefined ? {} : {isA}),
+    ...(newType === 'list') ? {data: {}} : {},
+    newType: adjustedNewtype,
+    ...(pattern === undefined ? {} : {pattern}),
```

Comment:
This works well enough, but looks like code that was updated in a previous PR.

Reply:
done. Thanks for pointing it out!